src/HashedInner.hs

```diff
@@ -242,7 +242,7 @@ topologicalSort expr@(mp, n) = filter (/= -1) . UA.elems $ topoOrder
             return order
 
 -- | Modification will return an ExpressionDiff instead of the whole Expression to speed things up
---
+-- What ExpressionDiff stands for?
```

Comment:
I think it would be better to completely write what Expression Difference is doing so other members would easily find out have they should work with it.

Reply:
What do you mean "what Expression Difference is doing"? It's represents the data and it doesn't do anything else. It represents the difference between the updated expression and the orgininal expression. It is the combination of `extraEntries` and `newRootId` which are commented